pkg/cli/environment.go

```diff
@@ -119,6 +137,7 @@ func (s *EnvSettings) EnvVars() map[string]string {
 		"HELM_KUBECONTEXT":   s.KubeContext,
 		"HELM_KUBETOKEN":     s.KubeToken,
 		"HELM_KUBEAPISERVER": s.KubeAPIServer,
+		"HELM_MAX_HISTORY":   strconv.Itoa(s.MaxHistory),
```

Comment:
Move just after `HELM_NAMESPACE` because comment applies to the kube vars

Reply:
Feeling your feedback, done.